### ISSUE TITLE
Request - Store usernames in lowercase / case-insensitive

### ISSUE THREAD

**Issue by ghost** [bludit/bludit#1225]
Hellow,

### Describe your problem
The current version allows the creation of case-sensitive user accounts. Thus, "editor", "EDITOR", "editOR" can exist in parallel with each other, which not only leads to problems when logging in, but also leads to confusing mix-ups and generally provides a rather unsightly environment.

The most content mangement systems doesn't allow them, mostly because the most database systems are working case-insensitive per default, of course.

So, it would be really awesome when the key, which is used on the users.php database file, is stored and checked in lowercase only. This way Bludit could avoid multiple user accounts with the same name.

~ Sam.

**Comment by dignajar** [comment]
It's possible to store the users in lowercases and fix this problem.
https://github.com/bludit/bludit/issues/1225

**Comment by dignajar** [comment]
I roll-back because will break some installations, I will implement for the v4.0

**Comment by dignajar** [comment]
Hello, with the new version of Bludit v4.0 rc1, I would like to close the old Github issues. If you feel that your issue is not resolved in the latest version, create a new ticket.

- Help and Support use the Forum https://forum.bludit.org
- Bugs and new requests here in Github https://github.com/bludit/bludit/issues
